package.json

```diff
@@ -125,7 +124,7 @@
     "babel-plugin-transform-inline-environment-variables": "^0.4.4",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "babel-plugin-transform-replace-expressions": "^0.2.0",
-    "chai": "^4.5.0",
+    "chai": "^5.2.0",
```

Comment:
Bumping chai here might not be trivial without first doing it in the monorepo. 🤔 🙈 
https://github.com/mui/mui-x/pull/17575